src/components/profile/ProfileMenu.tsx

```
@@ -22,7 +22,7 @@ export default function ProfileMenu({ onMenuClick }: ProfileMenuProps) {
   const pathname = usePathname();
   const router = useRouter();
   const [isUploading, setIsUploading] = useState(false);
-  const [imageUrl, setImageUrl] = useState<string>('/icons/profile_default.svg');
+  const [imageUrl, setImageUrl] = useState<string>('/imgs/profile_default.png');
```

ESLint

[error] 'imageUrl' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)